app/cmd/root.go

```diff
@@ -77,6 +77,9 @@ func addCommonLintAndInstallFlags(flags *pflag.FlagSet) {
 	flags.StringSlice("chart-repos", []string{}, heredoc.Doc(`
 		Additional chart repos to add so dependencies can be resolved. May be
 		specified multiple times or separate values with commas`))
+	flags.StringSlice("helm-repo-extra-args", []string{}, heredoc.Doc(`
+		Additional arguments for helm repo add. (e.g. --username test --password secret)
+		May be specified multiple times or separate values with commas`))
```

Comment:
Looking at this again, we should make clear how to configure it, i. e. with `--helm-repo-extra-args <repo>=<args>`. An example in the readme would be great as well.

Reply:
How about this:

```go
flags.StringSlice("helm-repo-extra-args", []string{}, heredoc.Doc(`
	Additional arguments for the 'helm repo add' command to be 
	specified on a per-repo basis with an equals sign as delimiter 
	(e. g. 'myrepo=--username test --password secret'). May be specified
	multiple times or separate values with commas`))
```